voorhoede/tudelft-spacefinder
# Feat/highlight selected space marker

# Changes


[feat: add highlighted marker on map](https://github.com/voorhoede/tudelft-spacefinder/pull/204/commits/17452377c4d09852157ec5eb5adddf1fe063b509)
[feat: add purple marker circle](https://github.com/voorhoede/tudelft-spacefinder/pull/204/commits/90724a9019d78c4cad6bb2a74b07897738ddfdae)
[feat: zoom in to building on cluster click](https://github.com/voorhoede/tudelft-spacefinder/pull/204/commits/bede568f354ae149086f9161d3fff38e35b1b04b)
[feat: also zoom to building or space on refresh or on click](https://github.com/voorhoede/tudelft-spacefinder/pull/204/commits/89e089166b870c91c9c32472f3fc876dbdf87d84)
[feat: also add marker on refresh of space slug page](https://github.com/voorhoede/tudelft-spacefinder/pull/204/commits/cecd7b40ca6fbabb044620447e5d68482b3811d8)
[feat: add onboarding to ignore](https://github.com/voorhoede/tudelft-spacefinder/pull/204/commits/5d48cc917abd054b62222790c65b8a3437fbc96f)
[feat: change to zoom level 18 for spaces](https://github.com/voorhoede/tudelft-spacefinder/pull/204/commits/f5528c4eadd7dce0b8804fe1d099aa74e9cdc3b6)
[refactor: zoom levels and icon sizes](https://github.com/voorhoede/tudelft-spacefinder/pull/204/commits/ed3b26c5aa765a264f18686f92e63c18eaa34fe7)

# Associated issue


Resolves https://trello.com/c/DoZXIhad/100-highlight-selected-space-marker

# How to test



1. Open preview link
2. Navigate on the map single point gets highlighted
3. Check if highlight is there on refresh or sharing page
4. Adjusted zoomlevels on navigating between buildings

# Checklist
